Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.16.5-buster AS build-env
+
+# Install minimum necessary dependencies,
+ENV PACKAGES make git gcc
+RUN apt-get update -y
+RUN apt-get install -y $PACKAGES
+
+# Create directory
+RUN mkdir -p /src/panacea-core /src/wasmvm
+
+# Add 'panacea-core' source files
+COPY . /src/panacea-core
+
+# Set working directory for the 'panacea-core' build
+WORKDIR /src/panacea-core
+
+# Install panacea-core
+RUN make clean && make build
+
+# Get 'libwasmvm.so' from wasmvm
+RUN git clone -b v0.14.0 https://github.com/CosmWasm/wasmvm.git /src/wasmvm
```

Comment:
It is same version used by wasmd(v0.17.0)
If change the wasmd version, wasmvm version needs to be update too.